### ISSUE TITLE
Ability to have a heating and cooling emission system

### ISSUE THREAD

**Issue by pjayathissa** [architecture-building-systems/RC_BuildingSimulator#10]
@Mifehr: You designed an emission system where you can choose a heating and cooling emission system. However in the app it will only call the heating system, not the cooling.

Can this code
```
 heatingEmissionSystem=NewRadiators,
 coolingEmissionSystem=AirConditioning,
```

Be simplified to `EmissionSystem=NewRadiators` and assume that the heating and cooling system share the same emission system?

Cheers

PJ

**Comment by Mifehr** [comment]
@pjayathissa
Yes, this should be possible. As long as the temperature demanded by the emission system is correct, the cooling supply system can use the same emission system. The reason why this was split is that some emission systems aren't really used for both heating and cooling (e.g. condensation on floor).

**Comment by pjayathissa** [comment]
However, in the app, it doesn't actually run the cooling system, just the heating system

```python
def calc_heat_flow(self,T_out, internal_gains, solar_gains, energy_demand):
		#C.1 - C.3 in [C.3 ISO 13790]

		#Calculates the heat flows to various points of the building based on the breakdown in section C.2, formulas C.1-C.3
		#Emission System Director is called to action (setBuilder and calcFlows available)
		emDirector = EmissionDirector()
		
		
		emDirector.setBuilder(self.heatingEmissionSystem(T_out=T_out, internal_gains=internal_gains, solar_gains=solar_gains, energy_demand=energy_demand, mass_area=self.mass_area, A_t=self.A_t, h_tr_w=self.h_tr_w, T_set_heating=self.T_set_heating, T_set_cooling=self.T_set_cooling))  #heatingEmissionSystem chosen
		
		flows = emDirector.calcFlows()
		
		self.phi_ia = flows.phi_ia 
		self.phi_st = flows.phi_st 
		self.phi_m = flows.phi_m 
		self.heatingSupplyTemperature = flows.heatingSupplyTemperature
		self.coolingSupplyTemperature = flows.coolingSupplyTemperature
```

**Comment by pjayathissa** [comment]
closed with PR #45
